docs/developer/docker.md

```diff
@@ -0,0 +1,13 @@
+# Developing the docker environment
+
+We use docker and docker-compose in production, and it can also be used as a local development environment.
+
+If you make changes to either `Dockerfile` or `docker-compose.yml` you'll need to rebuild it locally to test it:
+
+```
+$ docker-compose down // (if running)
+$ docker-compose build --no-cache
+$ docker-compose up // (to restart)
+```
```

Comment:
Not a blocker, but if you're like me and copy and paste commands annotating them with "#" is useful as it means that the shell will treat it as a comment as well. 
```bash
docker-compose down # (if running) cat /dev/urandom ;)
```